requirements.txt

```diff
@@ -1,10 +1,10 @@
-bayesian-changepoint-detection>=0.2.dev1
+git+https://github.com/hildensia/bayesian_changepoint_detection@2dd95f5c1d028116899a842ccb3baa173f9d5be9
```

Comment:
Nit: We might want to add a comment to this line to make it easier on why we are doing this.

Reply:
Added!